examples/provider/provider.tf

```diff
@@ -3,5 +3,5 @@ provider "sigsci" {
   email = "" // Required. may also provide via env variable SIGSCI_EMAIL
   //  auth_token = "" // May also provide via env variable SIGSCI_TOKEN
   //  password = ""   // May also provide via env variable SIGSCI_PASSWORD
-  //  fastly_key = "" // May also provide via env variable FASTLY_KEY. Required for Edge Deployments functionality.
+  //  fastly_key = "" // May also provide via env variable FASTLY_API_KEY. Required for Edge Deployments functionality.
```

Comment:
@ponkio-o should this be `fastly_api_key`?

Reply:
You're right. Thanks!